android/src/main/java/com/swmansion/reanimated/NativeProxy.java

```diff
@@ -141,8 +165,13 @@ private void setGestureState(int handlerTag, int newState) {
   }
 
   @DoNotStrip
-  private String getUpTime() {
-    return Long.toString(SystemClock.uptimeMillis());
+  private String getUptime() {
+    if (slowAnimationsEnabled) {
+      long ANIMATIONS_DRAG_FACTOR = 10;
```

Comment:
make it `private final static`

Reply:
It's inlined anyway. I added `final` for a nice touch.